crypto/evp/keymgmt_meth.c

```diff
@@ -267,6 +270,19 @@ void EVP_KEYMGMT_names_do_all(const EVP_KEYMGMT *keymgmt,
         evp_names_do_all(keymgmt->prov, keymgmt->name_id, fn, data);
 }
 
+const OSSL_PARAM *EVP_KEYMGMT_gettable_params(const EVP_KEYMGMT *keymgmt)
+{
+    return evp_keymgmt_gettable_params(keymgmt);
+}
+const OSSL_PARAM *EVP_KEYMGMT_settable_params(const EVP_KEYMGMT *keymgmt)
+{
+    return evp_keymgmt_settable_params(keymgmt);
+}
+const OSSL_PARAM *EVP_KEYMGMT_gen_settable_params(const EVP_KEYMGMT *keymgmt)
+{
+    return evp_keymgmt_gen_settable_params(keymgmt);
+}
```

Comment:
Is there a reason for these wrappings instead of just renaming the functions in question?

Reply:
Yes...
I got stuck in a quandry because you internally documented them.
And then it looks funny in that doc to have a evp_keymgmt_get but not the equivalent lowercase gettable. Would you prefer it to do See the uppercase variant in that doc?

Reply:
If they no longer exist in their internal form, I would simply remove them from internal documentation.